source/_enterprise.html.md.erb

```diff
@@ -4,6 +4,21 @@ The following section describes the specific event types and payloads returned f
 
 With Event Reporting, you can programmatically retrieve events that are occurring in your Smartsheet organization account. Examples of events are create, update, load, and delete of sheets, reports, dashboards, attachments, and users. Note: while the current list of events covers most of Smartsheet functionality, new events will be added to the list as Smartsheet introduces new functionality.
 
+Some actions in Smartsheet are performed by system components. Therefore, the userId property in some response payloads refers to a system userId and not to a regular user. For example, the Smartsheet Events API shows 5629504736520068 as the userId when an anonymous user accesses a sheet that is published for anyone. 
```

Comment:
Minor suggestion (just changed from`system userId` to `system user`):
the userId property in some response payloads refers to a system **user** and not to a regular user

Reply:
Thanks!